src/ui/server/content-strings/pages/insurance/eligibility/index.ts

```diff
@@ -180,4 +194,6 @@ export default {
   CONTRACT_TOO_SHORT,
   PARTY_TO_CONSORTIUM,
   MEMBER_OF_A_GROUP,
+  MEMBER_OF_A_GROUP_EXIT,
+  PARTY_TO_CONSORTIUM_EXIT,
```

Comment:
Not a blocker, but like this approach, would be nice to align other exit pages e.g changing `LONG_TERM_COVER` to `LONG_TERM_COVER_EXIT`, `NO_COMPANIES_HOUSE_NUMBER_EXIT`, `COMPANY_NOT_ACTIVE_EXIT`, etc

Reply:
tech debt for this? since the templates, strings, route names should change

Reply:
Created EMS-3690